src/core/proto_reader/reader.rs

```diff
@@ -248,7 +282,7 @@ mod test_proto_config {
         let reader = ProtoReader::init(ResourceReader::<Cached>::cached(runtime.clone()), runtime);
         let proto_no_pkg =
             PathBuf::from(tailcall_fixtures::configs::SELF).join("proto_no_pkg.graphql");
-        let config_module = reader.read(proto_no_pkg.to_str().unwrap()).await;
+        let config_module = reader.read(proto_no_pkg.to_str().unwrap(), None).await;
```

Comment:
Can we add tests for reading from proto path instead of default behavior.

Reply:
I’ve added tests to `tests/cli/fixtures/generator/gen_proto_with_proto_paths_config.md` that should cover my new code.